### PR TITLE
chore: add a Native version of the Dashboard app

### PR DESCRIPTION
**Pull request by PekoPPT**

### CODE REVIEW

File: examples-standalone/dashboard/src/components/IssuesComponent.vue [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+//
+// const token = '<personal access token>';
+//
+const token = ['6170ac11463601b547', '224777b801f2e889077ca9'].join('');
+
+export default {
+    components: {
+        "Grid": Grid,
+        custom: DetailComponent,
+        IDTemplate: markRaw(IDTemplate),
```
Comment (zdravkov) [inline examples-standalone/dashboard/src/components/IssuesComponent.vue:54]: do we need the markRaw function here?

File: examples-standalone/dashboard/src/main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { createApp } from 'vue'
+// Import our custom CSS
+import './scss/styles.scss'
+
+// Import all of Bootstrap's JS
+import * as bootstrap from 'bootstrap'
```
Comment (zdravkov) [inline examples-standalone/dashboard/src/main.js:6]: Do we need the Bootstrap's JS for anything?

**Review by zdravkov — APPROVED** [review]
Great job!

File: examples-standalone/dashboard/index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + Vue</title>
```
Comment (zdravkov) [inline examples-standalone/dashboard/index.html:7]: I was wandering if we could add here our icon and something that mentions Kendo

File: examples-standalone/dashboard/src/components/IssuesComponent.vue (hunk 1 above)
Comment (PekoPPT) [reply examples-standalone/dashboard/src/components/IssuesComponent.vue:54]: If the **markRaw** is not used, the following warning appears in the console: 
image

File: examples-standalone/dashboard/src/main.js (hunk 2 above)
Comment (PekoPPT) [reply examples-standalone/dashboard/src/main.js]: fixed

File: examples-standalone/dashboard/src/components/IssuesComponent.vue [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+            })
+    },
+    data: function () {
+        return {
+            loader: false,
+            response: [],
+            detailTemplate: 'DetailComponent',
+            expandedItems: [],
+            columns: [
+                { field: 'number', title: 'ID', width: 100, filterable: false, cell: IDTemplate },
```
Comment (zdravkov) [inline examples-standalone/dashboard/src/components/IssuesComponent.vue:84]: Oh I got it - here the cell should be a string - cell: 'IDTemplate' - and then we will be able to remove the above markRaw :)

Comment (PekoPPT) [reply examples-standalone/dashboard/src/components/IssuesComponent.vue]: WOW! Thank you :) Just fixed it.

File: examples-standalone/dashboard/index.html (hunk 3 above)
Comment (PekoPPT) [reply examples-standalone/dashboard/index.html]: I've updated the Meta title and changed the icon with one with a green Kendoka

**Comment by PekoPPT** [comment]
The thing we should do before officially presenting the application to the clients is to add the Badge component in several places in the code. The wrapper version of the Dashboard app uses image icons and Badge classes. These can be substituted for the Badge component. 

The next step is to wait till the beginning of September when the new Badge styles will be released. After that we can use the Badge component and apply it in the example.

**Pull request merged**